pkg/ssh/ssh.go

```diff
@@ -105,6 +105,59 @@ func (cfg *Config) addSSHFlag(s string) error {
 	return nil
 }
 
+func (cfg *Config) ApplyEnvironment(grafanaPDCEnvVarPrefix string) {
+	sshKeyFileEnvVar, ok := os.LookupEnv(fmt.Sprintf("%v_SSH_KEY_FILE", grafanaPDCEnvVarPrefix))
+	if ok && (cfg.KeyFile == DefaultConfig().KeyFile) {
+		cfg.KeyFile = sshKeyFileEnvVar
+	}
+
+	// Using the same logic as the `ssh.Config` definition, clamping to max of 3
+	sshLogLevelSelectEnvVar, ok := os.LookupEnv(fmt.Sprintf("%v_SSH_LOG_LEVEL", grafanaPDCEnvVarPrefix))
```

Comment:
This logic is great, but can we move this env override to the mainFlags ApplyEnvironment? The ssh-log-level is deprecated in favour of that one.